api/specs/api-v2.0.yaml

```
@@ -0,0 +1,1078 @@
+          format: email
+        company:
+          type: string
+          description: Nom de l'employeur de l'occupant
+        travel_pass:
+          type: object
+          description: |
+            Numéro de carte de transport (TCL, Navigo, Trabool, etc.) de l'occupant.
+            Obligatoire si l'information est disponible.
+            Actuellement supporté: Navigo
+          additionalProperties: false
+          required: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)